### PR TITLE
TP2000-401: Add test for chunk_taric

### PR DESCRIPTION
**Pull request by eadpearce**
# TP2000-401: Add test for chunk_taric

## Why

* We need to improve test coverage of the importer

## What

* This test covers previously uncovered lines of code

### CODE REVIEW

File: importer/tests/test_chunker.py [hunk 1]
```diff
@@ -115,3 +121,14 @@ def test_filter_transaction_records_negative(
...
     assert transaction is None
+
+
+def test_chunk_taric():
+    with open(f"{TEST_FILES_PATH}/goods.xml", "rb") as f:
+        content = f.read()
+    taric_file = SimpleUploadedFile("goods.xml", content, content_type="text/xml")
+    batch = factories.ImportBatchFactory.create()
+    chunk_taric(taric_file, batch)
+    assert ImporterXMLChunk.objects.count()
```
Comment (gabelton) [inline importer/tests/test_chunker.py:132]: For sanity checking, might be worth an assertion that there are no chunk objects prior to chunk_taric being called

File: importer/tests/test_chunker.py [hunk 2]
```diff
@@ -115,3 +121,14 @@ def test_filter_transaction_records_negative(
...
+    chunk = ImporterXMLChunk.objects.first()
+    assert chunk.chunk_text
```
Comment (gabelton) [inline importer/tests/test_chunker.py:134]: Do we know what the text should be ? can we add an assertion about it?

File: importer/tests/test_chunker.py [hunk 3]
```diff
@@ -115,3 +121,14 @@ def test_filter_transaction_records_negative(
     )
 
     assert transaction is None
+
+
+def test_chunk_taric():
```
Comment (gabelton) [inline importer/tests/test_chunker.py:126]: Could you add a docstring?

**Review by gabelton — APPROVED** [review]

File: importer/tests/test_chunker.py (hunk 2 above)
Comment (eadpearce) [reply importer/tests/test_chunker.py:134]: it should be the contents of the original xml file but it gets spaces removed, attributes reordered... so i gave up trying to test it

**Pull request merged**